### PR TITLE
Implementation of the Scheduler UI interface's base

### PR DESCRIPTION
**Pull request by niranjan94**
Implementation of the Scheduler UI interface's base as discussed here at #349 into the Server UI so that we can start working on it. Resolves issue #365 .

Demo: https://oes-testbed.codezero.xyz/scheduling/

The base interface contains:
- Drag and drop sessions into tracks
- Graphically resize sessions to change timings.

### CODE REVIEW

File: open_event/static/admin/js/scheduler.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
+/**
+ * Created by Niranjan on 23-May-16.
+ */
+(function ($) {
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l4)

Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l4)

File: open_event/static/admin/js/scheduler.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                pixels: 24
+            }
+        };
+
+        interact('.session')
+            .draggable({
+                // enable inertial throwing
+                inertia: false,
+                // enable autoScroll
+                autoScroll: false,
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l33)

File: open_event/static/admin/js/scheduler.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+                hours: 17,
+                minutes: 0
+            },
+            unit: {
+                minutes: 15,
+                pixels: 24
+            }
+        };
+
+        interact('.session')
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l28)

File: open_event/static/admin/js/scheduler.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+(function ($) {
+    $(document).ready(function () {
+        /**
+         *  24px == 15 minutes
+         *  (Smallest unit of measurement is 15 minutes)
+         *
+         */
+
+        // TIME SETTINGS
+        var time = {
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l13)

File: open_event/static/admin/js/scheduler.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+                hours: 9,
+                minutes: 0
+            },
+            end: {
+                hours: 17,
+                minutes: 0
+            },
+            unit: {
+                minutes: 15,
+                pixels: 24
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Unquoted property `pixels` found.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l24)

File: open_event/static/admin/js/scheduler.js [hunk 6]
```diff
@@ -0,0 +1,236 @@
+/**
+ * Created by Niranjan on 23-May-16.
+ */
+(function ($) {
+    $(document).ready(function () {
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Missing function expression name.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l5)

File: open_event/static/admin/js/scheduler.js [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+            start: {
+                hours: 9,
+                minutes: 0
+            },
+            end: {
+                hours: 17,
+                minutes: 0
+            },
+            unit: {
+                minutes: 15,
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l23)

File: open_event/static/admin/js/scheduler.js (hunk 5 above)
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l24)

File: open_event/static/admin/js/scheduler.js [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+         *
+         */
+
+        // TIME SETTINGS
+        var time = {
+            start: {
+                hours: 9,
+                minutes: 0
+            },
+            end: {
```
Comment (juslee) [inline open_event/static/admin/js/scheduler.js]: Codacy Issue found: [Expected indentation of 10 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2618824881/issues/source?bid=3233864&fileBranchId=3313342#l18)

**Comment by juslee** [comment]
This is ready for merging?

The scheduler looks good. Good job @niranjan94 and @SaptakS

**Comment by niranjan94** [comment]
@juslee yes. This is ready for merging. It needs to be merged to proceed with functionality additions.

**Comment by rafalkowalski** [comment]
@niranjan94 @juslee why to master branch? i think it should be gsoc2016

**Comment by niranjan94** [comment]
@rafalkowalski You're right. I'll create a new PR to merge into the `gsoc2016` branch instead. Sorry about that. :sweat_smile:

**Comment by rafalkowalski** [comment]
@niranjan94  i've prepared special place for you :) 
https://github.com/fossasia/open-event-orga-server/blob/gsoc2016/open_event/templates/gentelella/admin/event/details/scheduling.html

**Comment by niranjan94** [comment]
@rafalkowalski :smile: Thanks !

**Comment by niranjan94** [comment]
New PR opened at #375 for merging into the `gsoc2016` branch.

**Pull request closed without merging**